.github/workflows/docker_build.yaml

```diff
@@ -19,3 +19,21 @@ jobs:
           tags: uat:latest
       - name: Image digest
         run: echo ${{ steps.docker_build.outputs.digest }}
+
+  deploy_to_cfn:
+    runs-on: ubuntu-18.04 # for older version of awscli.
+    steps:
+      - id: file_changes
+        uses: trilom/file-changes-action@v1.2.3
+
+      - name: Checkout
+        if: contains(toJSON(steps.file_changes.outputs.files), 'infra/')
+        uses: actions/checkout@v2
+
+      - name: Run deploy-prod.
+        if: contains(toJSON(steps.file_changes.outputs.files), 'infra/')
```

Comment:
**Suggestion** (non-blocking): Might worth deploying even if we don't change the stack, i.e. push a new image?

Reply:
That's a good idea, but I think we want a much less invasive change there - just want to update the container.  Let me see what I can do there...

Reply:
Great, got it.